obelisk/python/obelisk_py/core/robot.py

```diff
@@ -60,10 +62,31 @@ def __init__(self, node_name: str) -> None:
         )
         self.register_obk_publisher(
             "pub_true_sim_state_setting",
-            key="publisher_true_sim_state",
+            key="pub_true_sim_state",
             msg_type=osm.TrueSimState,
             default_config_str="",
         )
+        self.shared_ctrl = None
+
+    def _set_shared_ctrl(self, ctrl: List[float]) -> None:
+        """Set the shared control array.
+
+        This is a convenience function to set the shared control array in a thread-safe manner. This should be called in
+        the apply_control function. The shared control array is used to communicate the control input to the simulator.
+
+        The values can be accessed in the simulator by running
+        ```
+        with self.lock:
+            ctrl = list(self.shared_ctrl)
```

Comment:
Can we change this to `ctrl_key` to match the C++ side and to emphasize that this attribute is a key?

Goes for all the member variables.